src/index.ts

```diff
@@ -255,7 +249,7 @@ DNS.prototype.getZones = function(query, callback) {
         return;
       }
 
-      var zones = arrify(resp.managedZones).map(function(zone) {
+      var zones = [...resp.managedZones].map(function(zone) {
```

Comment:
likewise. Half-hearted. I would have expected that you can drop the dependency on `arrify` if you are going to change this here, but `arrify` is still being used elsewhere.